src/frequenz/sdk/actor/_power_managing/_base_classes.py

```diff
@@ -49,14 +50,14 @@ class Report:
     target_power: Power | None
     """The currently set power for the batteries."""
 
-    inclusion_bounds: timeseries.Bounds[Power] | None
+    _inclusion_bounds: timeseries.Bounds[Power] | None
```

Comment:
I'd suggest not to use this terminology any longer, but if you decide to do so, it needs documentation.

Reply:
We'll be forced to change it when we upgrade to the new microgrid API.  Until then, the internal repr just reflects the API version.  Is why I made them internal anyway, so people have to deal with these terms anymore.